src/ContinuousEnergyNeutron.tex

```diff
@@ -109,29 +109,32 @@ \subsubsection{\textsf{ESZ} Block}\label{sec:ESZBlock}
   \end{tablenotes}
 \end{ThreePartTable}
 
-\subsubsection{\textsf{NU} Block}\label{sec:NUBlock}
-There are three possibilities for the \Block{NU}:
+\subsubsection{\textsf{NU} \& \textsf{DNU} Block}\label{sec:NUBlock}\label{sec:DNUBlock}
+
+The \Block{NU} is used to specify prompt and/or total \nubar and is present only if \jxs{2} > 0. Delayed \nubar data is specified in the \Block{DNU} (which is only present if \jxs{24} > 0) but it shares some of the tables defined in this section.
```

Comment:
Each time `\nubar` is used, there is no space after it in the PDF documentation. I think this needs to be
```suggestion
The \Block{NU} is used to specify prompt and/or total \nubar\ and is present only if \jxs{2} > 0. Delayed \nubar\ data is specified in the \Block{DNU} (which is only present if \jxs{24} > 0) but it shares some of the tables defined in this section.
```

Reply:
Consider it changed ;-)